### PR TITLE
Added disclaimer to the "add book form" as per the issue  #5585

### LINKED ISSUE #5585: Add book form does not have disclaimer.
The add book form does not present the following disclaimer: "By saving a change to this wiki, you agree that your contribution is given freely to the world under CC0. Yippee!"

By the time the patron sees this they have reached the end of the edit form AFTER the initial record has been created.

### Relevant url?
https://openlibrary.org/books/add

### Steps to Reproduce

1. Go to ...https://openlibrary.org/books/add
2. Do ...notice there is no disclaimer.

* Actual: 
* Expected: 

### Details

- **Logged in (Y/N)?** 
- **Browser type/version?** 
- **Operating system?** 
- **Environment (prod/dev/local)?** prod

### Stakeholders

@mekarpeles

### PR DESCRIPTION
**Pull request by sarathsajan**
Added disclaimer to the "add book form" as per this issue --> #5585

What issue does this PR close? 
Closes #5585

### CODE REVIEW

**Comment by jimchamp** [comment]
It looks like this PR includes some unwanted form elements:

confusing_ui

Ideally, the red box in the above picture would only contain the disclaimer and the "Add" and "Cancel" buttons.  This can be achieved by copying the disclaimer div from the EditButtons macro (`/openlibrary/macros/EditButtons.html`) to the add book template.

There should be a bit of space between the add book inputs and the element which contains the disclaimer and buttons.  This is already done in this template by the `<br>` element found before the "Add" button.  Placing the `<br>` before the disclaimer should work in this case.

Let me know if you have any questions about this, and I'll be happy to help!

File: openlibrary/templates/books/add.html [hunk 1]
```diff
@@ -68,6 +68,8 @@ <h1>$_("Add a book to Open Library")</h1>
                 <div class="label">$_("Since you are not logged in, please satisfy the reCAPTCHA below.")</div>
                 <div class="g-recaptcha" data-sitekey="$:recaptcha.public_key"></div>
             </div>
+        
+        $:macros.EditButtons(comment=work.comment_)
```
Comment (jimchamp) [inline openlibrary/templates/books/add.html:72]: This should be removed, and the disclaimer should be inside of the next div.

**Review by jimchamp — CHANGES_REQUESTED** [review]

File: openlibrary/templates/books/add.html (hunk 1 above)
Comment (sarathsajan) [reply openlibrary/templates/books/add.html:72]: Thanks for the screenshot showing the mistake. I have made a _new_ pull request #5604 with the mistakes corrected and taking your suggestions into consideration.

**Comment by jimchamp** [comment]
Closing this in favor of #5604.

**Pull request closed without merging**